website/docs/cloud-docs/policy-enforcement/index.mdx

```diff
@@ -23,7 +23,9 @@ You can use two policy-as-code frameworks to define fine-grained, logic-based po
 
 ## Apply Policy Sets to Workspaces and Projects
 
-Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set. 
+Policy sets are collections of policies you can apply globally or to specific [projects](/terraform/cloud-docs/workspaces/organize-workspaces-with-projects) and workspaces in your organization. For each run in the selected workspaces, Terraform Cloud checks the Terraform plan against the policy set.
+
+You can exclude workspaces from any global or project-scoped policy sets. A policy set's policies will not be enforced on any runs in an excluded workspace. Excluded workspaces override included workspaces, so even if a policy set is applied directly to a workspace, the workspace can still be excluded from the policy set at the project level.
```

Comment:
I don't _love_ the below suggestion, but I do think it makes it a little more clear. Specially, I think "excluded" v.s. "included" might be confusing to folks. What do you think?
```suggestion
You can also exclude specific workspaces from global or project-scoped policy sets. Terraform Cloud won't enforce a policy set's policies on any runs in an excluded workspace. For example, if you directly attach a policy set to a project and then exclude one of the project's workspaces from that policy set, Terraform Cloud will not enforce the policy set on that excluded workspace.
```

Reply:
I like it! I tweaked the suggestion a little bit in 25cdc9e.